extensions/amp-script/amp-script.md

```diff
@@ -77,63 +109,103 @@ You can also include your JavaScript inline, in a `script` tag. You must:
+* Property and attribute getters like `getAttribute()`, `hasAttribute()`
+* Event properties like `Event.target`, `Event.type`, and `Event.bubbles`
+* Element properties like `attributes`, `id`, `outerHTML`, `textContent`, `value`, `classList`, and `className`
+* And many more.
 
-`amp-script` supports getting and setting [`amp-state`](https://amp.dev/documentation/components/amp-bind/#initializing-state-with-amp-state) JSON via JavaScript.
+`querySelector()` is supported for simple selectors - element, ID, class, and attribute. So, `document.querySelector('.class')` will work, but `document.querySelector('.class1 .class2')` will not. [See the code](https://github.com/ampproject/worker-dom/blob/main/src/worker-thread/dom/Element.ts#L159) for details.
 
-This enables advanced interactions between `amp-script` and other AMP elements on the page via `amp-bind` [bindings](https://amp.dev/documentation/components/amp-bind/#bindings). Invoking `AMP.setState()` from `amp-script` may cause mutations to the DOM as long as it was triggered by user gesture, otherwise it will only implicitly set state (similar to [`amp-state` initialization](https://amp.dev/documentation/examples/components/amp-bind/?referrer=ampbyexample.com#initializing-state)).
+`amp-script` supports common Web APIs like `Fetch`, `Websockets`, `localStorage`, `sessionStorage`, and `Canvas`. Presently, the `History` API is not implemented, and neither are cookies.
+
+`amp-script` does not support the entire DOM API or Web API, as this would make `amp-script`'s own JavaScript too large and slow. If there's an API you'd like to see supported, please [file an issue](https://github.com/ampproject/amphtml/issues/new) or [suggest and contribute the change yourself](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md). For a complete list of supported DOM APIs, see the [API compatibility table](https://github.com/ampproject/worker-dom/blob/master/web_compat_table.md).
 
 [tip type="default"]
-`AMP.setState()` requires the [`amp-bind`](https://amp.dev/documentation/components/amp-bind) extension script to be included in the document head.
+For a set of samples showing `amp-script` in use, [see here](/documentation/examples/components/amp-script/).
 [/tip]
 
+### Frameworks and libraries
+
+Presently, libraries like [jQuery](https://jquery.com) will not work with `amp-script` without modification, as they use unsupported DOM APIs. However, [@ampproject/worker-dom](https://github.com/ampproject/worker-dom/) was designed to support the APIs that popular JavaScript frameworks use. `amp-script` has been tested with frameworks like [React](http://reactjs.org/), [Preact](https://preactjs.com/), [Vue](https://vuejs.org/), [Angular](https://angularjs.org/), [Aurelia](https://aurelia.io/), and [lit-html](https://lit-html.polymer-project.org/). To keep bundle sizes small, we recommend using Preact.
```

Comment:
I know that React/Preact have been tested, but are we sure the others work as expected? I remember there are some open issues about Vue support in the worker-dom repos.

Reply:
@kristoferbaxter wdyt? I included these because you told me you'd tested them! But we can certainly reduce this list. Just including React and Preact here is still nice.

Reply:
@kristoferbaxter I know things are a bit complicated right now, so if you can't get to this comment I can just remove the other frameworks for now. We can always add them back.

Reply:
Apologies for delays, I'd omit this segment for now and mention that we're working on active support for more and more frameworks.

Reply:
from offline conversation, I'll remove all but React and Preact.